src/main/java/com/opencart/steps/HeaderPageBL.java

```diff
@@ -68,18 +66,29 @@ public HeaderPageBL clickOnCurrencyButton(String currencyName) {
     }
 
     /**
-     * Method checks if MacBook Cost in specified currency is correct.
+     * Method finds all ProductContainers on the Page and checks
+     * if their Product Cost in specified currency is correct.
      *
-     * @param container is Container for Products where price needs to be checked;
      * @param currencyName is name of Currency that needs to be checked.
-     * @return HeaderPageBL
      */
 
-    public HeaderPageBL verifyMacBookCost(ProductContainer container, String currencyName) {
+    public void verifyAllProductsPrices(String currencyName) {
+        List<ProductContainer> productContainers = new SearchResultPage().getProductContainers();
+        for (ProductContainer container
+                : productContainers) {
+            verifyProductCost(container, currencyName);
+        }
+    }
 
+    /**
+     * Method checks if MacBook Cost in specified currency is correct.
+     *
+     * @param container    is Container for Products where price needs to be checked;
+     * @param currencyName is name of Currency that needs to be checked.
+     */
+    private void verifyProductCost(ProductContainer container, String currencyName) {
```

Comment:
How do you think is it a good approach to hardcode the real values of different currencies?
Imagine the situation when somebody goes to the Admin page and change it, your test will fail then.

So, I would recommend you do the next:
1. Go to the admin page and get real values from existing currencies, then add also add a new one by the autotest. 
2. Perform tests with different currencies including the newly created one.
3. You need to remove already created currency after your test finished.

Reply:
okay. 

Should I also go to AdminPage (screenshot) and read actual Product Prices from there?
and then compare them with existing ones on the website?

  <img width="1269" alt="Screen Shot 2020-11-20 at 13 09 15" src="https://user-images.githubusercontent.com/73054470/99793616-9948a980-2b31-11eb-9fbd-80553b155c88.png">